corehq/apps/dashboard/templates/dashboard/base.html

```diff
@@ -48,46 +48,46 @@ <h1>
                                     trigger: 'hover',
                                },
                            "></a>
-                        <span data-bind="text: title, visible: !url"></span>
-                    </div>
-                    <div class="panel-body">
+              <span data-bind="text: title, visible: !url"></span>
+            </div>
+            <div class="panel-body">
 
-                        <!-- Icon in a watermark style, displayed under item list -->
-                        <i data-bind="css: icon, visible: showBackgroundIcon" class="dashboard-icon dashboard-icon-medium dashboard-icon-bg"></i>
+              <!-- Icon in a watermark style, displayed under item list -->
+              <i data-bind="css: icon, visible: showBackgroundIcon" class="dashboard-icon dashboard-icon-medium dashboard-icon-bg"></i>
 
-                        <!-- Spinner -->
-                        <i class="fa fa-spinner fa-spin fa-5x" data-bind="visible: showSpinner"></i>
+              <!-- Spinner -->
+              <i class="fa fa-spinner fa-spin fa-5x" data-bind="visible: showSpinner"></i>
 
-                        <!-- List of items, with pagination if there are multiple pages -->
-                        <!-- ko if: hasItemList() -->
-                            <div data-bind="visible: showItemList">
-                                <div class="list-group" data-bind="foreach: items">
-                                    <div class="list-group-item"
-                                         data-bind="popover: {
+              <!-- List of items, with pagination if there are multiple pages -->
+              <!-- ko if: hasItemList() -->
```

Comment:
should this be indented too? Or will that happen in a different pass?
(cf: https://github.com/dimagi/commcare-hq/pull/24190#discussion_r280117365)

Reply:
I think pycharm is interpreting them as html comments which should be on the same indentation level. I'd vote for a separate pass (but I agree I think it could make this code confusing, so that's a soft vote and i'm easily convinced otherwise)

Reply:
another solution might be to skip those fiels that have html comments for now which looks to be a minority of these files

Reply:
i was going to go through on a separate pass. right now I'm just trying to get the majority of the code looking proper ASAP thru automation :-)